dev/bots/check_code_samples.dart

```diff
@@ -320,8 +320,6 @@ final Set<String> _knownMissingTests = <String>{
   'examples/api/test/material/platform_menu_bar/platform_menu_bar.0_test.dart',
   'examples/api/test/material/flexible_space_bar/flexible_space_bar.0_test.dart',
   'examples/api/test/material/selection_area/selection_area.0_test.dart',
-  'examples/api/test/material/app_bar/sliver_app_bar.2_test.dart',
-  'examples/api/test/material/app_bar/sliver_app_bar.3_test.dart',
```

Comment:
Looks like the **selection_area.0_test.dart** was just removed in the main branch, and since it was touching these deleted lines, GitHub is saying we have merge conflicts 😒